Project.toml

```diff
@@ -1,13 +1,13 @@
-authors = ["Samuel Omlin", "Ludovic Räss", "Ivan Utkin"]
+authors = ["Samuel Omlin", "Ludovic Raess", "Ivan Utkin"]
 name = "ImplicitGlobalGrid"
 uuid = "4d7a3746-15be-11ea-1130-334b0c4f5fa0"
-version = "0.12.0"
+version = "0.13.0"
 
 [compat]
-AMDGPU = "0.3.7"
-CUDA = "1, ~3.1, ~3.2, ~3.3, ~3.7.1, ~3.8, ~3.9, ~3.10, ~3.11, ~3.12, 4"
+AMDGPU = "0.5"
```

Comment:
Does it not anymore work with AMDGPU v0.3 or v4 after your latest changes? If it does, then you could do:
AMDGPU = "0.3.7, 0.4, 0.5"

Reply:
V0.5 is breaking with different (HIP) API, so unfortunately no, we cannot support previous versions here.